src/classes/HH_Container_TEST.cls

```diff
@@ -133,7 +133,6 @@ private class HH_Container_TEST {
 
     /*********************************************************************************************************
     * @description Tests deleteContacts()
-    */
     public static testMethod void testDeleteContacts() { 
```

Comment:
can't we just delete the method, or will it for sure be returning in the near future?